keras/layers/recurrent.py

```diff
@@ -560,6 +614,48 @@ def call(self, inputs, mask=None, training=None, initial_state=None):
         else:
             return output
 
+    def _normalize_args(self, inputs, initial_state, constants):
+        """When running a model loaded from file, the input tensors
```

Comment:
Nit: first line of docstring (one-line summary) should fit in one line and end with a period.